src/hooks/useMarkdownStyle.ts

```diff
@@ -21,7 +19,7 @@ function useMarkdownStyle(message: string | null = null): MarkdownStyle {
                 fontSize: variables.fontSizeLarge,
             },
             emoji: {
-                fontSize: emojiFontSize,
+                fontSize: inputContainsOnlyEmojis ? 27 : 19,
```

Comment:
Same, shouldn't we use already defined variables for those values, or create new ones if they are new?